### DISCUSSION TITLE
zh/guide/drivers/ilanzou

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#314] [Announcements]
# zh/guide/drivers/ilanzou

蓝奏云优享版：https://ilanzou.com 根文件夹ID 根目录ID，默认为0，其它目录ID查看下图获取方式 账户、密码 填写自己的蓝奏云优享版帐号密码 默认使用的下载方式

https://alist.nn.ci/zh/guide/drivers/ilanzou.html

**Comment by Masterain98** [comment]
上传文件时会随机地出现`-1: token不能为空！`错误

**Reply by ssyslk** [reply]
+1

**Comment by WhyWhySeeYa** [comment]
挂载存储没有对应的驱动方式吗

**Comment by izchuan** [comment]
已上传大于50M的视频就这样的错误提示  怎么回事额？
Request failed with status code 413

**Comment by songqingawa** [comment]
用一段时间之后突然变成了"failed get objs: failed to list objs: -1: 超时请求"怎么办

**Reply by zhoupingxiao** [reply]
我的也是

**Reply by ZOHI666** [reply]
+1

**Comment by sd362318** [comment]
failed get objs: failed to list objs: -1: 超时请求

**Comment by blueflower1** [comment]
failed get objs: failed to list objs: -1: 超时请求

**Comment by bilibili2077** [comment]
failed get objs: failed to list objs: -1: 超时请求

**Reply by blueflower1** [reply]
源码已经修复了哈，正式版的目前还没有更新，可以自己编译一份就没问题啦

**Comment by songqingawa** [comment]
failed link: failed get link: redirect failed, status: 200 怎么回事

**Reply by coolxitech** [reply]
大概是歇逼了

**Comment by Moebyte** [comment]
上传文件的时候，全部在根目录下，怎么解决啊

**Comment by SunCosmos** [comment]
failed link: failed get link: redirect failed, status: 200 该如何解决？

**Reply by coolxitech** [reply]
解决不了，等更新

**Comment by Banrion-m** [comment]
现在好像这个用不了了是不~下载显示200

**Comment by szkzn** [comment]
2024-09-10 GitHub 最新 Dev 版本，蓝奏云优享版可以用了

**Reply by szkzn** [reply]
可用的意思：可以添加 蓝奏云优享版，并正常下载
经测试，通过 Alist 上传到蓝奏云优享版的所有文件，必定返回 HTTP 状态码 524 错误，但是到蓝奏云优享版官网查看后台文件，部分文件几率上传成功，经过下载测试校验哈希，上传成功的文件没有损坏

**Comment by qdqqd** [comment]
波兰的serv00服务器挂载蓝奏云优享版的，报错 {"msg":"系统时间不正确","code":-2}

**Comment by Peter267** [comment]
Failed create storage in database: UNIQUE constraint failed: x_storages.mount_path

**Comment by a49942245a** [comment]
failed get objs: failed to list objs: -1: 请勿频繁请求 咋办？
